setup.py

```diff
@@ -32,7 +32,8 @@
     "sawmill >= 0.2.1, < 1",
     "six >= 1.15.0, < 2",
     "toml >= 0.10.1, < 1",
-    "ujson >= 2.0.3, < 4"
+    "ujson >= 2.0.3, < 4",
+    "versup >= 1.0.1",
```

Comment:
The dependency is not really necessary for the package installation, but then it doesn't really belong in "doc" and "test" either.

What do you think about adding it in an extra list that would be added to the "dev" extra requirement?

```python
DEV_REQUIRES = [
    "versup >= 1.0.1",
]

...

setup(
    ...
    extras_require={
        "doc": DOC_REQUIRES,
        "test": TEST_REQUIRES,
        "dev": DOC_REQUIRES + TEST_REQUIRES + DEV_REQUIRES
    },
   ...
)
```

Reply:
Yes, ideally it should be a dev dependency, but I didn't want to add a new section. I'll make that change

Reply:
Looks good, could you remove "versup" from the installation requirements?

Also could you add an extra blank line between `DEV_REQUIRES` and `BuildExtended`? 
https://www.python.org/dev/peps/pep-0008/#blank-lines

As discussed previously, this is where [black](https://github.com/psf/black) would definitely be useful! (#36)